src/code/dialogs/checklist.js

```diff
@@ -51,12 +54,24 @@ const OperationChecklistDialog = WDialog.extend({
   _displayDialog: async function () {
     const operation = getSelectedOperation();
     loadFaked(operation);
+
+    // defaults to sorting by op order
+    if (localStorage[this.SORTBY_KEY] == null) {
+      localStorage[this.SORTBY_KEY] = 0;
+    }
+    if (localStorage[this.SORTASC_KEY] == null) {
+      localStorage[this.SORTASC_KEY] = "true";
+    }
```

Comment:
what do you think about moving this logic to `sortable.js` instead, using the constructor (for instance) (or a method) to set those keys and default value ?

Reply:
This suggestion would work, but it would not allow the owner dialog the chance to choose a default other than column 0 and ascending.

Reply:
you can set the default value and the storage keys in a single step, ignoring the default value if there are already value in localStorage

Reply:
Yes, this does clean up the change quite a bit. Good idea!

Reply:
I tried this, but it is problematic as we are passing sortBy and sortAsc from localStorage through functions before the default is set in the SortAscStoreKey or SortByStoreKey, which is called after a new sortable is created. One way to fix it, is to never set the sortBy and sortAsc outside of sortable, and just set the StoreKeys. This will clean up the code even more, as we don't need any of the dialogs to even care about sortBy and sortAsc. It is all handled by sortable if the StoreKeys are set.